gamemode/load_assets.lua

```diff
@@ -0,0 +1,16 @@
+local function runRecurse( dir, ext, f )
+    local files, dirs = file.Find( dir .. "/*", "GAME" )
+    if not files then return end
+    for k, v in pairs( files ) do
```

Comment:
Can we use a more descriptive variable name for `v`

Reply:
aye, ill clean this func up a lil